migrations/20220830011729_group_configs__using_group_roles__add.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE group_configs ADD COLUMN using_group_roles boolean DEFAULT false;
+
+ALTER TABLE group_users DROP CONSTRAINT group_users_pkey;
+ALTER TABLE group_users ADD PRIMARY KEY (group_id, user_id, group_role_id);
```

Comment:
To maintain integrity, you should do these in a single statement:

```suggestion
ALTER TABLE group_users
DROP CONSTRAINT group_users_pkey,
ADD PRIMARY KEY (group_id, user_id, group_role_id);
```

@mwest1066 could I get your eyes on this too?